front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -239,8 +252,8 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   });
 
   useEffect(() => {
-    const trainUsedForProjection = projectPathTrainResult.find(
-      (train) => train.id === selectedProjectionId
+    const trainUsedForProjection = projectPathTrainResult.find((train) =>
+      train.id.includes(selectedProjectionId)
```

Comment:
Why did you change this ?
```suggestion
      train.id === selectedProjectionId
```

Reply:
becaue the projected train can be paced or trainschedule, here we have occurrence and trainsschedule, so to handle the paced case i use includes

Reply:
My bad, you're right :+1: